packages/CodeQuality/tests/Rector/Array_/CallableThisArrayToAnonymousFunctionRector/Fixture/no_return_for_void.php.inc

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace Rector\CodeQuality\Tests\Rector\Array_\CallableThisArrayToAnonymousFunctionRector\Fixture;
+
+class HasVoidMethod
+{
+    public function run(array $values)
+    {
+        usort($values, [$this, 'compareSize']);
```

Comment:
This should be "thisReturnsVoid" I guess

Reply:
Yes, you are right: fixed!